TKeazirian.HTTPServer.Tests/Response/ResponseTests.cs

```diff
@@ -10,15 +10,16 @@ public class ResponseTests
     [Fact]
     public void BuildNewResponseBuildsNewResponseObject()
     {
-        string responseStatus = Constants.Status200;
-        string responseHeaders = HelperFunctions.CreateTestResponseHeaders();
-        string responseBody = "Hello world";
+        const string responseStatus = "HTTP/1.1 200 OK";
+        var responseHeaders = HelperFunctions.CreateTestResponseHeaders();
+        const string responseBody = "Hello world";
```

Comment:
why were these changed to `const` `string`? Constants are typically declared at a class level. did you mean for them to be `static`?

Reply:
This is just what the Rider IDE said - it was giving me the green underline squiggly and then prompted me to do this.  So I figured it was the most correct?

Reply:
it seems odd to me. I'd just ignore the squigglies as unless there is a more compelling reason, this seems odd.